### ISSUE TITLE
[BUG] AIWin keyboard focus feedback can be very difficult to discern when a high contrast theme is active 

### ISSUE THREAD

**Issue by guybark** [microsoft/accessibility-insights-windows#817]
1. Turn on the Windows High Contrast White theme. 
2. Run AIWin and tab around the UI.
3. Verify that the keyboard focus feedback shown in AIWin has strong contrast against its surroundings.

The image below shows that the keyboard focus feedback on the More Options button is a black border around a dark purple fill. I probably wouldn't have noticed it at all if the border hadn't slightly protruded into some white surroundings.

![image](https://user-images.githubusercontent.com/14315611/84696406-4d019980-af01-11ea-93fc-a8723dc360e4.png)

 - OS: Windows 10 1909
 - Accessibility Insights for Windows Version: 1.1.1248.1

**Comment by adarshrema** [comment]
Color overlay in HC white mode may be causing the issue. Needs more investigation to figure out the best fix.

**Comment by DaveTryon** [comment]
@guybark, this has been fixed in the release available at https://github.com/microsoft/accessibility-insights-windows/releases/tag/v1.1.1388.01. Please validate/close at your convenience. Thanks!

**Comment by guybark** [comment]
I do think the keyboard focus feedback is improved, as there's a more noticeable change as keyboard focus moves to and from the Options button.  I do think when looking at the UI when keyboard focus is transitioning, it's still not clear where keyboard focus is still not clear, as shown in the screenshot below. But I'll close this anyway given the improvement.

![image](https://user-images.githubusercontent.com/14315611/97212838-25a7bc80-177e-11eb-9377-66d9de843a60.png)
